studies/tasks.py

```diff
@@ -479,17 +479,24 @@ def build_framedata_dict(filename, study_uuid, requesting_user_uuid):
 
 
 @app.task(bind=True)
-def delete_video_from_cloud(task, s3_video_name, recording_method_is_pipe):
+def delete_video_from_cloud(
+    task, s3_video_name, recording_method_is_pipe, study_type_is_jspsych
+):
     """Delete videos in S3.
 
     Meant to have a delay of about 7 days.
     """
-    if recording_method_is_pipe:
-        # delete from Pipe bucket
-        S3_RESOURCE.Object(settings.BUCKET_NAME, s3_video_name).delete()
+    if study_type_is_jspsych:
+        # delete from lookit-jspsych bucket
+        S3_RESOURCE.Object(settings.JSPSYCH_S3_BUCKET, s3_video_name).delete()
     else:
-        # delete from RecordRTC bucket
-        S3_RESOURCE.Object(settings.S3_BUCKET_NAME, s3_video_name).delete()
+        # EFP
+        if recording_method_is_pipe:
```

Comment:
This is the most unnecessary comment, but you could use an `elif` here instead.

Reply:
Not an unnecessary comment - thanks for noticing and flagging this!

I wrote this with `elif` the first time, then re-wrote it this way to make the logic clearer. The properties `recording_method_is_pipe` and `recording_method_is_recordrtc` both return `False` (not null/undefined) for jsPsych studies, so I was (overly) worried that this could lead to bugs if we ever write the conditionals in a different way - e.g. trying to use `not recording_method_is_pipe/recordrtc` to differentiate EFP types, which would also catch jsPsych studies. But I'm now realizing that that would just be a super weird way of doing things and it's not something I should worry about 🤦‍♀️ Anyway, I'm going to re-write it.